### ISSUE TITLE
[bitnami/postgresql-ha] postgresql.username ignored when creating new database

### ISSUE THREAD

**Issue by jrtcppv** [bitnami/charts#1702]
**Which chart**: 

postgresql-ha

**Description**

Setting the postgresql.username field 

**Steps to reproduce the issue:**

1. Install chart with `postgresql.username` set to something other than `postgres`
2. Get a shell into a postgresql pod: `kubectl exec -it <name of pod> -- /bin/bash`
3. Try to start psql with expected creds: 
```
PGPASSWORD=$POSTGRES_PASSWORD psql -U $POSTGRES_USERNAME -d $POSTGRES_DB
```
This will result in a password authentication failure.
4. Try to start psql with default username `postgres`:
```
PGPASSWORD=$POSTGRES_PASSWORD psql -U postgres -d $POSTGRES_DB
```
psql will start successfully.

**Describe the results you received:**

Failure to log in by pg_pool and inability to log in manually via psql

**Describe the results you expected:**

Database created with expected username

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.0.0-beta.5", GitCommit:"509a8819b7e8dba86b7982c006fce22628aeea28", GitTreeState:"clean", GoVersion:"go1.13.3"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.3", GitCommit:"5e53fd6bc17c0dec8434817e69b04a25d8ae0ff0", GitTreeState:"clean", BuildDate:"2019-06-06T01:44:30Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.8", GitCommit:"211047e9a1922595eaa3a1127ed365e9299a6c23", GitTreeState:"clean", BuildDate:"2019-10-15T12:02:12Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by alemorcuq** [comment]
Hi, @jrtcppv.

I'm also having issues to login if I set an user with the `postgresql.username` parameter, so I will have to look deeper into this. However, I'm not able to login as the `postgres` user either, unlike you.

**Comment by eserdk** [comment]
Also facing the issue when neither postgresql.username nor postgresql.password params are not working. 
But it's possible to login as postgres user cause postgres-password is generated automatically when database is instantiated - check your kubectl secrets for this. @alemorcuq @jrtcppv

**Comment by alemorcuq** [comment]
I've created a PR fixing the issue. Turns out we were using a wrong variable for the database username. The issue should be fixed when #1770 and #1772 are merged.

cc @eserdk
